packages/data-explorer/src/components/dataView/DefaultCardContent.vue

```diff
@@ -1,6 +1,19 @@
 <template>
   <div>
     <h5 class="card-title mg-default-card-title">{{dataLabel}}
+      <a
+        class="btn btn-sm btn-link ml-1"
+        role="button"
+        :href="'/plugin/data-row-edit/' + dataTable + '/' + dataId">
```

Comment:
Add "plugin" to devServer proxy to be able to test the changes locally

Reply:
o cool does work ? i will try it , good suggestion